app/server/models/transfer_card_usage.py

```diff
@@ -21,3 +21,11 @@ class TransferCardUsage(ModelBase):
     # Credit transfer
     credit_transfer_id       = db.Column(db.Integer, db.ForeignKey("credit_transfer.id"), index=True)
     credit_transfer          = db.relationship('CreditTransfer', foreign_keys=[credit_transfer_id], backref=backref('transfer_card_usage', uselist=False), lazy='select')
+
+    session_number = db.Column(db.Integer, default=-1)
+    amount_loaded = db.Column(db.Integer, default=-1)
+    amount_deducted = db.Column(db.Integer, default=-1)
+    
+    @hybrid_property
+    def balance(self):
```

Comment:
This is going to be used later to calculate the difference between the transfer account balance, and the last known card balance